ArduPlane/mode_loiter.cpp

```diff
@@ -58,7 +58,7 @@ bool ModeLoiter::isHeadingLinedUp_cd(const int32_t bearing_cd)
     */
 
     // Use integer division to get discrete steps
-    const int32_t expanded_acceptance = 1000 * (labs(plane.loiter.sum_cd) / 36000);
+    const int32_t expanded_acceptance = 1000 * (labs(plane.loiter.sum_cd) * 0.00002777f; // 0.00002777 = divide by 36,000
```

Comment:
Floating point problems, plus out of scope with the above comment, needs a bit more looking at the usage and less mechanical replacement.